tests/test_data_cloud.py

```diff
@@ -565,3 +572,41 @@ def test_error(self):
         self.main_fail(['push', f])
         self.main_fail(['pull', f])
         self.main_fail(['fetch', f])
+
+
+class TestWarnOnOutdatedStage(TestDvc):
+    color_patch = patch.object(logger, 'colorize', new=lambda x, color='': x)
+
+    def main(self, args):
+        ret = main(args)
+        self.assertEqual(ret, 0)
+
+    def _should_test(self):
```

Comment:
No need for this. This method is used in remote tests, since not every remote type might be available. Local remote is always available though :)

Reply:
EDIT: I meant `_should_test` method.